nvtabular/ops/categorify.py

```diff
@@ -753,13 +775,17 @@ def _encode(name, storage_name, path, gdf, cat_cache, na_sentinel=-1, freq_thres
         ).sort_values("order")["labels"]
         labels.fillna(na_sentinel, inplace=True)
         labels = labels.values
-    else:
-        # Use `searchsorted` if we are using a "full" encoding
+    if search_sorted:
```

Comment:
Can this just be an else still - just to make it clear that we're doing one branch or the other?